core/client/app/controllers/posts.js

```diff
@@ -74,7 +74,7 @@ export default Controller.extend({
     postListFocused: equal('keyboardFocus', 'postList'),
     postContentFocused: equal('keyboardFocus', 'postContent'),
 
-    sortedPosts: computed('model.@each.status', 'model.@each.publishedAt', 'model.@each.isNew', 'model.@each.updatedAt', function () {
+    sortedPosts: computed('model.@each.status', 'model.@each.publishedAtOffset', 'model.@each.isNew', 'model.@each.updatedAtOffset', function () {
```

Comment:
I think you'll need to stick with `publishedAt` rather than `publishedAtOffset` here because `publishedAtOffset` is an async property. For sorting it shouldn't matter as we'll always be dealing with a list that's in chronological order, the timezone offset won't have any effect.